service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -209,6 +215,29 @@ private void sendCMOCaseLinkNotifications(final EventData eventData) {
         sendCMOCaseLinkNotificationToRepresentatives(eventData);
     }
 
+    @EventListener
+    public void sendNotificationToPartiesAddedToCase(PartyAddedToCaseEvent event) {
+        List<Element<Representative>> representatives = event.getRepresentativesToNotify();
+        EventData eventData = new EventData(event);
+
+        representatives.stream().forEach(representativeElement -> {
+            String email = representativeElement.getValue().getEmail();
+            RepresentativeServingPreferences servingPreferences
+                        = representativeElement.getValue().getServingPreferences();
+            if (servingPreferences != POST) {
+
+                Map<String, Object> parameters = partyAddedToCaseContentProvider
+                    .getPartyAddedToCaseNotificationParameters(event.getCallbackRequest().getCaseDetails(),
+                           servingPreferences);
+                String template = partyAddedToCaseContentProvider
+                    .getPartyAddedToCaseNotificationTemplate(servingPreferences);
+
+                sendNotification(template, email, parameters,
+                        eventData.getReference());
+            }
+        });
```

Comment:
I think that some minor formatting fixes might improve the readability of this method. 

```
        representatives.stream().forEach(representativeElement -> {
            String email = representativeElement.getValue().getEmail();
            RepresentativeServingPreferences servingPreferences
                        = representativeElement.getValue().getServingPreferences();

            if (servingPreferences != POST) {
                Map<String, Object> parameters = partyAddedToCaseContentProvider
                    .getPartyAddedToCaseNotificationParameters(event.getCallbackRequest().getCaseDetails(),
                           servingPreferences);

                String template = partyAddedToCaseContentProvider
                    .getPartyAddedToCaseNotificationTemplate(servingPreferences);

                sendNotification(template, email, parameters,
                        eventData.getReference());
            }
        });
```

Reply:
i like this :)